main.py

```diff
@@ -145,6 +145,8 @@ def compress_tar(self, path, target):
         files = os.listdir(path)
         tar = tarfile.open(target, 'w')
         for file in files:
+            if file == ".DS_Store":
```

Comment:
whichever you think is best! I just did .DS_Store to limit the scope of the PR in case I missed any cases where dotfiles ended up being important.